doc/source/io.rst

```diff
@@ -4633,6 +4635,41 @@ Read only certain columns of a parquet file.
    os.remove('example_pa.parquet')
    os.remove('example_fp.parquet')
 
+
+Handling Indexes
+''''''''''''''''
```

Comment:
not really sure you need this doc section at all. ``index=False`` is a pretty common option in pandas.

Reply:
True, but the goal of this section is mostly to illustrate use cases and how engines differ.